easybuild/easyconfigs/f/Flye/Flye-2.7-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'Flye'
+version = '2.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/fenderglass/Flye'
+description = """Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
+ and Oxford Nanopore Technologies."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
```

Comment:
Doesn't look like there is any MPI or LAPACK used in this. Can it be dropped to iccifort?